apps/website/src/content/docs/keyboardkeys.mdx

```diff
@@ -1,18 +1,27 @@
 ---
 title: Keyboard keys
-description:
-propsPath: '@itwin/itwinui-react/esm/core/Typography/Kbd.d.ts'
+description: This component is used to display which key or combination of keys performs a given action.
 thumbnail: Keyboard
 ---
 
 <p>{frontmatter.description}</p>
 
-<Placeholder componentPageName='keyboard-key' />
-
 <LiveExample src='Keyboard.main.tsx' truncate={false}>
   <AllExamples.KeyboardMainExample client:load />
 </LiveExample>
 
+## Usage
+
+By default, the `<Kbd>` component renders as a [`<kbd>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/kbd) HTML element. You can either pass custom key(s) as the `children` or use the `KbdKeys` enum which includes a convenient set of constants for common keyboard keys such as `KbdKeys.Command`, `KbdKeys.Enter`, and `KbdKeys.Right`.
```

Comment:
For readability, we should split the `KbdKeys` part into a separate paragraph and possibly list out all values available.

Reply:
could you also split the codeblock into two? first showing a basic example and then showing one with `KbdKeys`

Reply:
yes. by "split" i also mean moving them into the proper positions: paragraph, then codeblock which relates to that paragraph, then another paragraph, then another codeblock which relates to that paragraph.